tests/unit/legacy/api/test_json.py

```diff
@@ -30,14 +31,9 @@
 
 
 def _assert_has_cors_headers(headers):
-    assert headers["Access-Control-Allow-Origin"] == "*"
-    assert headers["Access-Control-Allow-Headers"] == (
-        "Content-Type, If-Match, If-Modified-Since, If-None-Match, "
-        "If-Unmodified-Since"
-    )
-    assert headers["Access-Control-Allow-Methods"] == "GET"
-    assert headers["Access-Control-Max-Age"] == "86400"
-    assert headers["Access-Control-Expose-Headers"] == "X-PyPI-Last-Serial"
+    for k, v in _CORS_HEADERS.items():
+        assert k in headers
+        assert headers[k] == v
```

Comment:
I would guess the reason for hardcoding is that it is easy to modify `warehouse.utils.cors:_CORS_HEADERS` and have the test still passing.

I'm happy with this change, but maybe the core devs will prefer to make sure the test comparison does not rely on data in the main codebase.